### DISCUSSION TITLE
I have some questions regarding certain features.

### DISCUSSION THREAD

**Discussion by lastingwind** [microsoft/pyright#9290] [General]
I am using Theia to create a web-based Python editor and want to use Pyright. My business requirement is to read the contents of a txt file and treat all the contents of this txt file as variables in my Python editor without reporting errors.Can Pyright accomplish this functionality?If it cannot, are there any Theia plugins that can achieve this functionality?

**Comment by erictraut** [comment]
Since this isn't a bug report, I'm moving it to the discussions section.
